src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -17,3 +18,27 @@
   {:background-color (colors/theme-colors colors/neutral-5 colors/white-opa-5)
    :border-width     1
    :border-color     (colors/theme-colors colors/neutral-20 colors/white-opa-20)})
+
+(defn message-default-style
+  []
+  {:font-family    "Inter-Regular"
```

Comment:
I think we should use `foundations/typography` here and remove the duplicated properties. Like this:

```clojure
(defn message-default-style
  []
  (assoc typography/paragraph-1 :color (colors/theme-colors colors/neutral-100 colors/white)))
```

Reply:
yes there shouldn't be any custom styles